packages/neo4j-driver/test/examples.test.js

```diff
@@ -708,7 +708,7 @@ describe('#integration examples', () => {
   it('service unavailable example', done => {
     const console = consoleOverride
     const consoleLoggedMsg = consoleOverridePromise
-    const uri = `bolt://${sharedNeo4j.hostname}:7688` // wrong port
+    const uri = `bolt://${sharedNeo4j.hostname}:7686` // wrong port
```

Comment:
:eyes: why?

Reply:
The `7688` port is the server side routing port. Since fabric is enabled by default in 5.0, this error is not happening consistently. So I move it to an unused port.